tests/test_linear_sync.py

```
@@ -23,6 +23,19 @@
 
 import pytest
 
+# Check if gql is available for tests that need it
+try:
+    import gql  # noqa: F401
```

Ruff (0.14.14)

[warning] Unused `noqa` directive (non-enabled: `F401`)

Remove unused `noqa` directive

(RUF100)